Files/ViewModels/ItemViewModel.cs

```diff
@@ -717,10 +717,14 @@ public async Task ReloadItemGroupHeaderImagesAsync()
                     foreach (var gp in FilesAndFolders.GroupedCollection)
                     {
                         var img = await GetItemTypeGroupIcon(gp.FirstOrDefault());
-                        await CoreApplication.MainView.DispatcherQueue.EnqueueAsync(() =>
+                        if (img is byte[])
                         {
-                            gp.Model.ImageSource = img;
-                        }, Windows.System.DispatcherQueuePriority.Low);
+                            gp.Model.ImageBytes = img as byte[];
```

Comment:
Bound to UI Tag, crashes with wrong thread exception when list is grouped by file type.

Reply:
@gave92 Actually, do we support icons for the header groups? If not, then I'd like to remove this altogether.

Reply:
Yes, if you group by file type and click on a header to go to the zoomed out view